File: api/src/main/java/com/datastrato/graviton/exceptions/NotFoundException.java
/*
* Copyright 2023 Datastrato.
* This software is licensed under the Apache License version 2.
*/

package com.datastrato.graviton.exceptions;

public class NotFoundException extends GravitonRuntimeException {

  public NotFoundException(String message) {
    super(message);
  }

  public NotFoundException(String message, Throwable cause) {
    super(message, cause);
  }
}

File: clients/client-python/gravitino/exceptions/not_found_exception.py
"""
Copyright 2024 Datastrato Pvt Ltd.
This software is licensed under the Apache License version 2.
"""

from gravitino.exceptions.gravitino_runtime_exception import GravitinoRuntimeException


class NotFoundException(GravitinoRuntimeException):
    """Base class for all exceptions thrown when a resource is not found."""
